profile/2019-03-01-hybrid.json

```diff
@@ -6,175 +6,472 @@
     "resource-manager": {
         "microsoft.authorization": {
             "2016-09-01": [
-                "locks"
+                {
+                    "resourceName": "locks",
+                    "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/resources/resource-manager/Microsoft.Authorization/stable/2016-09-01/locks.json"
```

Comment:
our Initial design goal for this profile format  is to get all the info in this file from  ARM itself.  I am not opposed to adding a path here , if it is going to be useful.

Reply:
Hi! 
Including this path would break the Azure Gov profile, which I will be releasing by end of day. We have coined two profiles before without the path required and check against the spec repo before placing it in this repo. Is this path needed? How has profile generation occurred before prior to the path? @shahabhijeet

Reply:
@sijuman this profile format never worked for us. Plus this is an additive changes and should not affect the model for whoever is consuming.
From day one, the piece of feedback to your team was that this format DOES NOT work.
But due to urgency for your team to release, we went ahead with it and worked with the old format.
All languages in this repository will use the new format, so that is the reason the new format should be default.
But as I said, as this is an additive change, it should not affect anything, plus the new format makes it clear on which resourceType to use and is the spec in public domain.
You should ask gov. team to start using the new format.